src/windows/title_editor.py

```diff
@@ -586,83 +530,52 @@ def update_background_color_button(self):
             # Store the opacity as the color's alpha level
             color.setAlphaF(opacity)
             self.bg_color_code = color
+            log.debug("Set color of background-color button to %s", color.name())
 
     def set_font_style(self):
         '''sets the font properties'''
         # Loop through each TEXT element
         for text_child in self.text_node + self.tspan_node:
             # set the style elements for the main text node
             s = text_child.attributes["style"].value
-            # split the text node so we can access each part
-            ar = s.split(";")
-            # we need to find each element that we are changing, shouldn't assume
-            # they are in the same position in any given template.
-
-            # ignoring font-weight, as not sure what it represents in Qt.
-            fs = self.find_in_list(ar, "font-style:")
-            ff = self.find_in_list(ar, "font-family:")
-            fp = self.find_in_list(ar, "font-size:")
-            if fs:
-                ar[fs] = "font-style:" + self.font_style
-            if ff:
-                ar[ff] = "font-family:" + self.font_family
-            if fp:
-                ar[fp] = "font-size:%spx" % self.font_size_pixel
-            # rejoin the modified parts
-            t = ";"
-            self.title_style_string = t.join(ar)
+            ard = style_to_dict(s)
+            set_if_existing(ard, "font-style", self.font_style)
+            set_if_existing(ard, "font-family", self.font_family)
+            set_if_existing(ard, "font-size", f"{self.font_size_pixel}px")
```

Comment:
Oh, yes, purely by coincidence this PR also introduces the first [f-string](https://realpython.com/python-f-strings/) info the OpenShot codebase, making it officially require Python 3.6 or later to run.

It's time. Python 3.5 has [actually been end-of-life since September](https://www.python.org/dev/peps/pep-0478/). And f-strings are just too damn convenient _not_ to use!